react.gradle

```diff
@@ -110,7 +110,8 @@ afterEvaluate {
                 execCommand.addAll([*nodeExecutableAndArgs, cliPath])
             }
         } else {
-            execCommand.addAll([npx, "react-native"])
+            throw new Exception("Missing cliPath or nodeExecutableAndArgs from build config. " +
+                "Please set set project.ext.react.cliPath to the path of the react-native cli.js");
```

Comment:
Thank you for taking your time to improve this experience! Do you think we should mention `android/app/build.gradle` explicitly in the error message so it is clear where the config is located?

Reply:
You're correct. It's probably worth adding. I was simply following the convention on line 49:

```
throw new Exception("OS not recognized. Please set project.ext.react.hermesCommand " +
                        "to the path of a working Hermes compiler.");
```

If we include the path in this exception message then it needs to be added to the one on line 49.

Reply:
I'm OK merging this as is, since it follows convention. I'm not as familiar with Gradle, so my main worry is whether these error messages are clear to the average Android developer. Would they know where to go to fix this config? Does the config location vary across workspaces or is it generally expected at the same path?

In other words, if the config is always at `android/app/build.gradle` then perhaps we can explicitly point to that file.